app/src/main/java/app/gamenative/ui/screen/xserver/XServerScreenUtils.kt

```
@@ -0,0 +1,150 @@
+                            intArrayOf(i), false,
+                            object : IArchiveExtractCallback {
+                                override fun getStream(index: Int, extractAskMode: ExtractAskMode): ISequentialOutStream? {
+                                    if (extractAskMode != ExtractAskMode.EXTRACT) return null
+
+                                    return ISequentialOutStream { data ->
+                                        fos.write(data)
+                                        data.size // Return the number of bytes written
+                                    }
+                                }
+
+                                override fun prepareOperation(extractAskMode: ExtractAskMode) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)